packages/types/src/shared.ts

```diff
@@ -649,106 +649,106 @@ export enum ApplicationCommandPermissionTypes {
 }
 
 /** https://discord.com/developers/docs/topics/permissions#permissions-bitwise-permission-flags */
-export enum BitwisePermissionFlags {
+export const BitwisePermissionFlags = {
```

Comment:
Actually now that we switched to a const, why not this to make sure that next time Discord add a new permission string we don't forget to update this or the PermissionStrings type?
```suggestion
export const BitwisePermissionFlags: Record<PermissionStrings, bigint> = {
```

Reply:
1. the as const is useless at that point
2. PermissionStrings is `keyof typeof BitwisePermissionFlags`, so it becomes a dependency loop in the type checker